estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name':'estate',
+    'version':'1.0',
+    'author':'ksni-odoo',
+    'depends':[
+        'base'
+    ],
+    'installable':True,
+    'application':True,
+    'data':[
+        'data/ir.model.access.csv',
+        'views/estate_property_views.xml',
+        'views/estate_property_type_views.xml',
+        'views/estate_property_tag_views.xml',
+        'views/estate_property_offer_views.xml',
```

Comment:
```suggestion
        'views/estate_property_offer_views.xml',
        'views/estate_property_tag_views.xml',
        'views/estate_property_type_views.xml',
```
We should load `estate_property_offer_views` before `estate_property_type_views` as we are calling an action which is defined in offer_views from type_views